ocaml/src/alba.ml

```diff
@@ -435,23 +435,31 @@ let alba_show_object
        >>= fun (hm,r) ->
        match r with
          | None ->
-            Lwt_io.eprintlf "not found"
+            Lwt.fail_with "not found"
```

Comment:
this path doesn't take the to-json flag into account...
Otoh if we add a note about the to-json flag for this command being for dev/test purposes only then it doesn't have to